torch/_prims/vjp.py

```diff
@@ -0,0 +1,217 @@
+# Module for defining vector-Jacobian product (VJP) functions for PrimTorch
```

Comment:
Should we define the vjps alongside the prims themselves? Seems like a fundamental transform

Reply:
Maybe we should. I did that initially in https://github.com/pytorch/pytorch/pull/80448 by adding `impl_vjp=` alongside prims.